frontend/routes/manage/Questionnaire.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+    <div>
+        <div class="space-between">
+            <h1>Responses</h1>
+            <div>
+                <router-link to="./message"><Button content="Message"></Button></router-link>
+                <TextInput placeholder="Search bar..."></TextInput>
+            </div>
+        </div>
+        <Table :rows="tableRows" :shownColumns="tableColumns"></Table>
```

Comment:
This needs `:showEditLink="true"` to show the edit icon. It also needs an event handler for navigating to the edit page; you can look at [my schedule page work in `he-794`](https://github.com/codeRIT/hackathon-manager/blob/he-794/frontend/routes/manage/schedule/Schedule.vue) as an example.